tools/depends/native/TexturePacker/src/XBTFWriter.cpp

```diff
@@ -116,12 +117,15 @@ bool CXBTFWriter::UpdateHeader(const std::vector<unsigned int>& dupes)
   {
     CXBTFFile& file = files[i];
 
-    // Convert path to lower case
+    // Convert path to lower case and store it into a fixed size array
     std::string path = file.GetPath();
+    char pathMem[CXBTFFile::MaximumPathLength];
+    memset(pathMem, 0, sizeof(pathMem));
+
     for (std::string::iterator ch = path.begin(); ch != path.end(); ++ch)
```

Comment:
We need a memory which has a size of 256 bytes and contains the path. I'm not sure why it was done this way but that's how the old texturepacker worked so we need to keep that for backwards compatibility.